app/controllers/admin/volunteers.py

```diff
@@ -32,11 +32,7 @@ def trackVolunteersPage(eventID):
         abort(404)
 
     program = event.singleProgram
-
-    # TODO: What do we do for no programs or multiple programs?
-    if not program:
-        return "TODO: What do we do for no programs or multiple programs?"
-
+    
     trainedParticipantsList = trainedParticipants(program, g.current_term)
```

Comment:
Does this function handle `program` being `None`? Also, there are two cases to consider: events with no programs and events with multiple programs (probably only the All Volunteer Training)